logging/main.tf

```diff
@@ -0,0 +1,224 @@
+#VCN loggroup resource
```

Comment:
Since this file defines all the log groups, it should be named as such

Reply:
I have included the logsource in the main.tf as suggested by cetin when creating a separate module.Since this file also has the module call i kept it as main.tf